Library/Homebrew/cmd/help.rb

```diff
@@ -52,10 +53,26 @@ def help_for_command(cmd)
     end
     return if cmd_path.nil?
 
-    cmd_path.read.
+    text = cmd_path.read.
       split("\n").
       grep(/^#:/).
-      map { |line| line.slice(2..-1).delete("`").sub(/^  \* /, "brew ") }.
+      map { |line| line.slice(2..-1).sub(/^  \* /, "#{Tty.highlight}brew#{Tty.reset} ") }.
       join("\n")
+
+    return text.delete("`") unless $stdout.tty?
+
+    i = 0
+    while text.include? "`"
+      if (i+=1) % 2 == 1
+        text.sub!("`", Tty.highlight)
+      else
+        text.sub!("`", Tty.reset)
+      end
+    end
```

Comment:
I would prefer if this was done with a regular expression and `gsub`. I think the code will be shorter and easier to understand.